DeusEx/Classes/DXRando.uc

```diff
@@ -1,4 +1,5 @@
-class DXRando extends Info;
+class DXRando extends Info
+    transient;

```

Comment:
for some reason ReEntry stopped working, making DXRando transient fixed it but I don't know why this issue came out of nowhere, I probably copied a file from the repo to my deus ex folder that I haven't updated in a while?